### PR TITLE
Currently all tests are failing for 8.8.x branch.

### PR DESCRIPTION
**Pull request by JTubex**

### CODE REVIEW

**Comment by JTubex** [comment]
Strange that DomainGetResponseTest is failing for Drupal 8.7 and PHP 7.1 and Drupal 8.8 and PHP 7.2. But at least the tests are running again.

**Comment by JTubex** [comment]
hmm, I guess this is a random failure since I ran the tests again and now it's failing for Drupal 8.7 PHP 7.1 and PHP 7.2.

**Comment by JTubex** [comment]
All tests are passing again.

File: .travis.yml [hunk 1]
```diff
@@ -7,13 +7,13 @@ services:
...
   fast_finish: true
   include:
-    - env: DRUPAL=8.7.x
+    - env: DRUPAL_INSTALL="drush dl drupal-8.7.x --drupal-project-rename=drupal" UPDATE_PHPUNIT="composer run-script drupal-phpunit-upgrade"
       php: 7.1
-    - env: DRUPAL=8.7.x
+    - env: DRUPAL_INSTALL="drush dl drupal-8.7.x --drupal-project-rename=drupal" UPDATE_PHPUNIT="composer run-script drupal-phpunit-upgrade"
       php: 7.2
-    - env: DRUPAL=8.8.x
+    - env: DRUPAL_INSTALL="composer create-project drupal/legacy-project:8.8.0-beta1 drupal" UPDATE_PHPUNIT=""
```
Comment (andypost) [inline .travis.yml]: would be great to update to rc1 https://www.drupal.org/project/drupal/releases/8.8.0-rc1

Comment (JTubex) [reply .travis.yml]: Done.

**Comment by agentrickard** [comment]
Sorry, I've been working on other things and just saw this. DomainGetResponseTest  has always had intermittent failures on Travis, and I don't know why.

**Pull request closed without merging**